src/logging/Manager.cc

```diff
@@ -750,12 +804,22 @@ bool Manager::Write(EnumVal* id, RecordVal* columns_arg)
 			auto v = filter->policy->Invoke(columns, IntrusivePtr{NewRef{}, id},
 			                                IntrusivePtr{NewRef{}, filter->fval});
 			if ( v && ! v->AsBool() )
+				{
+				// Hrml... not sure what's right. Seems not counting
+				// vetoes that are redundant is somewhat reasonable.
```

Comment:
No intuition either, but the comment sounds like there should be some discussion happening on this at some point. Is there a ticket or discussion attached to it? Than a link to that would help. Otherwise, I'd drop the comment or reword it to just explain the rationale (e.g. "Don't count redundant vetoes to have the metric reflect unique vetoes.").

Reply:
I've dropped the conditional counting and removed the comment. I think the unusual metric behavior can be explained by the not-quite-intuitive behavior that if a veto happened anyway on a stream level, we still ask the filter whether it would veto, too.